src/main/java/com/meta/cp4m/ServiceConfiguration.java

```diff
@@ -19,26 +19,33 @@ public class ServiceConfiguration {
   private final String handler;
   private final @Nullable String store;
   private final String plugin;
+  private final @Nullable String[] preProcessors;
 
   @JsonCreator
   ServiceConfiguration(
       @JsonProperty("webhook_path") String webhookPath,
       @JsonProperty("handler") String handler,
       @JsonProperty("store") @Nullable String store,
-      @JsonProperty("plugin") String plugin) {
+      @JsonProperty("plugin") String plugin,
+      @JsonProperty("pre_processors") @Nullable String[] preProcessors) {
```

Comment:
in general in Java you don't uses arrays unless you have a good reason too, typically you use a `List`